modules/luci-mod-network/htdocs/luci-static/resources/view/network/interfaces.js

```diff
@@ -678,6 +678,16 @@ return view.extend({
 						so = ss.taboption('general', form.Value, 'leasetime', _('Lease time'), _('Expiry time of leased addresses, minimum is 2 minutes (<code>2m</code>).'));
 						so.optional = true;
 						so.default = '12h';
+						so.validate = function (section_id, value) {
+							if (value === "infinite" || value === "deprecated") {
+								return true;
+							}
+
+							if (!/^[0-9]+[smhdwSMHDW]?$/.test(value)) {
```

Comment:
I think we should n[ew RegExp](https://github.com/search?q=repo%3Aopenwrt%2Fluci%20RegExp&type=code) its more common in LuCI

Reply:
I have modified the code to use `new RegExp` as suggested.